.github/workflows/run-tests.yml

```diff
@@ -29,17 +36,10 @@ jobs:
           path: ~/.composer/cache/files
           key: dependencies-laravel-${{ matrix.laravel }}-php-${{ matrix.php }}-composer-${{ hashFiles('composer.json') }}
 
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php }}
-          extensions: pdo, sqlite, pdo_sqlite
-          coverage: none
-
       - name: Install dependencies
         run: |
           composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-          composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
```

Comment:
Any particular reason why you removed this?

Reply:
As that is deprecated in Composer v2
https://getcomposer.org/upgrade/UPGRADE-2.0.md

Reply:
Ah, good catch! I didn't know that